mscore/mixer/mixer.cpp

```diff
@@ -76,7 +76,7 @@ char userRangeToReverb(double v) { return (char)qBound(0, static_cast<int>(v / 1
 
 Mixer::Mixer(QWidget* parent)
     : QDockWidget(qApp->translate("Mixer", "Mixer"), parent),
-      showDetails(true),
+      showDetails(preferences.getBool(PREF_IO_MIDI_SHOWCONTROLSINMIXER)),
```

Comment:
Somehow this doesn't seem to result in the prefs being read and set when opening the Mixer for the 1st time